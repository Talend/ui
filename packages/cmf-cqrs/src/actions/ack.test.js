import {
	addContext,
	receiveMessage,
	deleteACK,
} from './ack';

describe('actions.ack.addContext', () => {
	it('should return action', () => {
		const action = addContext(null, {
			requestId: '123',
			data: { foo: 'bar' },
			actionCreator: 'my super action creator id',
		});
		expect(action).toMatchSnapshot();
	});
});

describe('actions.ack.receiveMessage', () => {
	it('should return action', () => {
		const action = receiveMessage(null, {
			requestId: '123',
		});
		expect(action).toMatchSnapshot();
	});
});


describe('actions.ack.deleteACK', () => {
	it('should return action', () => {
		const action = deleteACK(null, {
			requestId: '123',
		});
		expect(action).toMatchSnapshot();
	});
});
