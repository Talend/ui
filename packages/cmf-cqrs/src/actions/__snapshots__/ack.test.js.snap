// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`actions.ack.addContext should return action 1`] = `
Object {
  "actionCreator": "my super action creator id",
  "data": Object {
    "foo": "bar",
  },
  "requestId": "123",
  "type": "ACK_ADD_CONTEXT",
}
`;

exports[`actions.ack.deleteACK should return action 1`] = `
Object {
  "requestId": "123",
  "type": "ACK_DELETE",
}
`;

exports[`actions.ack.receiveMessage should return action 1`] = `
Object {
  "requestId": "123",
  "type": "ACK_RECEIVE_MESSAGE",
}
`;
